UserApi/UserApi/Helper/UserProfileHelper.cs

```diff
@@ -58,17 +65,17 @@ public async Task<UserProfile> GetUserProfile(string filter)
             return response;
         }
 
-        private List<string> GetUserCaseTypes(List<AdGroup> userGroups)
-        {
-            return userGroups.Where(IsCaseType).Select(g => g.ToString()).ToList();
-        }
+        private static bool IsCaseType(AdGroup group) => CaseTypeMappings.ContainsKey(group);
 
-        private bool IsCaseType(AdGroup group)
+        private static List<string> GetUserCaseTypes(IEnumerable<AdGroup> userGroups)
```

Comment:
Return IEnumerable as the caller just needs IEnumerable, remove the ToList() in this method

Reply:
Fixed